src/common.rs

```diff
@@ -318,6 +319,35 @@ impl SectionId {
         }
     }
 
+    /// Returns the XCOFF section name for this kind.
+    #[cfg(target_os = "aix")]
+    pub fn name(self) -> &'static str {
+        match self {
+            SectionId::DebugAbbrev => ".dwabrev",
+            SectionId::DebugAddr => panic!("XCOFF doesn't have debug address section"),
```

Comment:
We can't panic like this. Instead, the `xcoff_name` method should return `Option<&'static str>`.